h/api/models/elastic.py

```diff
@@ -117,7 +117,7 @@ def shared(self):
         elif len(read_perms) > 1 and self.groupid == '__world__' and gperm in read_perms:
             read_perms = [gperm]
 
-        if (read_perms != [gperm] and read_perms != [self.userid]):
+        if (gperm not in read_perms and read_perms != [self.userid]):
```

Comment:
> The alternative would be to fix those tests, either to just use ["group:wibble"] or to patch the model code.

If the existing tests don't match real data that exists in Elasticsearch, then yes, let's fix the tests.